x/leveragelp/spec/incentivized_liquidation_system.md

```diff
@@ -0,0 +1,44 @@
+|            |                  |            |                  |             |                
+| Depositors |                  | Borrowers  |                  | Liquidators |                
+|            |                  |            |                  |             |                  
++-----+------+                  +-----+------+                  +-----+-------+               
+      |                               |                               |                             
+      | Deposit stable coins          | Borrow stable coins           | Check positions                
+      | and earn interest             | to open leverage positions    | for liquidation or if certain price is met(stop loss)     
+      |                               |                               |                              
++-----v------+                  +-----v------+                  +-----v-------+                   
+|            |                  |            |                  |             |                 
+|  Stable    |<-----------------| Leverage   |<-----------------| Liquidation |
+|  Coin Pool |   Borrow coins   | Position   |   Monitor        |   Function  |   Monitor        
+|  Module    |                  | Module     |   positions      |             |   positions       
++------------+                  +------------+                  +-------------+                  
+      |                               |                               |                               
+      |                               |                               |                                
+      |  Interest                      | Position health calculated   | If health < threshold, or certain price is met(stop loss)     
+      |                               | as lpAmount / debt            | trigger liquidation           
+      |                               |                               | and receive reward             
+      |                               |                               |                              
++-----v------+                  +-----v------+                  +-----v-------+                  
+|            |                  |            |                  |             |                   
+|  Depositor |                  |  Borrower  |                  |  Liquidator |                   
+|  Earns     |                  | Manages    |                  | Earns       |                  
+|  Interest  |                  | Position   |                  | Reward      |                  
++------------+                  +------------+                  +-------------+                   
+```
+
+## Explanation
+
+- **Depositors** provide liquidity to the system and earn interest.
+- **Borrowers** take leveraged positions by borrowing from the pool.
+- **Liquidators** and **Bots** monitor positions and execute liquidations when necessary.
```

Comment:
we might wanna explain here what we mean by `Bots`

Reply:
ok